### ISSUE TITLE
no such table: auth

### ISSUE THREAD

**Issue by Junyu25** [tsudoko/anki-sync-server#30]
I followed the instruction of this [bolg](http://genedan.com/no-127-ankisyncd-a-custom-sync-server-for-anki-2-1/), and install all the dependencies, But still come out of this error when I want to create a new user. How can I fix this?
```bash
sudo python3 ./ankisyncctl.py adduser test
Enter password for test:    
Traceback (most recent call last):   
  File "./ankisyncctl.py", line 82, in <module>   
    main()   
  File "./ankisyncctl.py", line 74, in main   
    cmds[c](arg)   
  File "./ankisyncctl.py", line 25, in adduser   
    user_manager.add_user(username, password)   
  File "/home/junyu/anki-sync-server/ankisyncd/users.py", line 108, in add_user   
    self._add_user_to_auth_db(username, password)   
  File "/home/junyu/anki-sync-server/ankisyncd/users.py", line 125, in _add_user_to_auth_db   
    (username, pass_hash))   
sqlite3.OperationalError: no such table: auth   
```

**Comment by tsudoko** [comment]
My bad, pushed a fix in b9a1203491c85e47a2843f4ac87086a2693e8320. Remove `auth.db` and `session.db`, run `git pull` in the main server directory and try again.

**Comment by Junyu25** [comment]
> My bad, pushed a fix in [b9a1203](https://github.com/tsudoko/anki-sync-server/commit/b9a1203491c85e47a2843f4ac87086a2693e8320). Remove `auth.db` and `session.db`, run `git pull` in the main server directory and try again.

Thanks, Problem solved.

**Comment by genedan** [comment]
I actually just installed the latest version about a week ago, after having gone 2-3 months without updating. Some changes that I'll need to update accordingly would be:

- No longer have to move the modules
- No longer have to configure ankisyncd
- Code in __init.py__ that goes into the addons folder has changed
